docker-compose.yml

```diff
@@ -1,12 +1,18 @@
-dist:
-  image: maxcnunes/electron-distribution:eb3node4-onbuild
-  # Build all and pack only for Windows
-  # because is not possible packing for OSX from Linux.
-  # Use unsafe-perm to force npm respect the tmp path (http://git.io/vB2oR)
-  command: npm run --unsafe-perm dist:winlinux
-  environment:
-    # Keep the tmp folder inside the project.
-    # This way we dont lose some cache files between distribution tasks
-    TMPDIR: /usr/src/app/.tmp
-  volumes:
-    - .:/usr/src/app
+version: '3'
```

Comment:
Are you still using this file for anything? Should it be removed?

Reply:
It was used for builds when we just had Ubuntu OS in the CI. But, it maybe makes sense keeping it around just in case we need to test a build locally for other OS than our own host OS.